token-lending/program/src/instruction.rs

```diff
@@ -914,3 +952,41 @@ pub fn set_lending_market_owner(
         data: LendingInstruction::SetLendingMarketOwner { new_owner }.pack(),
     }
 }
+
+/// Creates a `FlashLoan` instruction.
+#[allow(clippy::too_many_arguments)]
+pub fn flash_loan(
+    program_id: Pubkey,
+    amount: u64,
+    reserve_liquidity_pubkey: Pubkey,
+    destination_liquidity_pubkey: Pubkey,
+    reserve_pubkey: Pubkey,
+    lending_market_pubkey: Pubkey,
+    derived_lending_market_authority: Pubkey,
+    flash_loan_receiver_pubkey: Pubkey,
+    flash_loan_fees_account_info: Pubkey,
+    host_fee_recipient: Pubkey,
+    flash_loan_receiver_program_params: Vec<Pubkey>,
+) -> Instruction {
+    let mut accounts = vec![
+        AccountMeta::new(reserve_liquidity_pubkey, false),
+        AccountMeta::new(destination_liquidity_pubkey, false),
+        AccountMeta::new_readonly(reserve_pubkey, false),
+        AccountMeta::new_readonly(lending_market_pubkey, false),
+        AccountMeta::new_readonly(derived_lending_market_authority, false),
+        AccountMeta::new_readonly(flash_loan_receiver_pubkey, false),
+        AccountMeta::new_readonly(spl_token::id(), false),
+        AccountMeta::new(flash_loan_fees_account_info, false),
+        AccountMeta::new(host_fee_recipient, false),
+    ];
+    accounts.extend(
+        flash_loan_receiver_program_params
```

Comment:
These aren't params, they are account pubkeys.

Reply:
Changed to `flash_loan_receiver_program_account_pubkeys`.